project/plugins.sbt

```diff
@@ -2,6 +2,12 @@
 //
 // e.g. addSbtPlugin("com.github.mpeltonen" % "sbt-idea" % "1.1.0")
 
+// Add EPFL Artifactory if on CI
+resolvers ++= {
```

Comment:
Also you're not adding it to all the scripted tests (sbt-dotty/sbt-test/...), is that needed to?